cmd/performance-profile-creator/cmd/root.go

```diff
@@ -36,8 +36,6 @@ import (
 
 var (
 	validTMPolicyValues = []string{kubeletconfig.SingleNumaNodeTopologyManagerPolicy, kubeletconfig.BestEffortTopologyManagerPolicy, kubeletconfig.RestrictedTopologyManagerPolicy}
-	// TODO: Explain the power-consumption-mode and associated kernel args
-	validPowerConsumptionModes = []string{"default", "low-latency", "ultra-low-latency"}
```

Comment:
why is this gone?

Reply:
This was moved to profilecreator pkg.

Reply:
ok, so this was a dupe I guess.

Reply:
Yes, we forgot to remove it in a previous PR. I combined it with a smaller set of changes where we add logs to show system info. But will keep in mind for the future to have changes like this in a completely separate commit.